commands/initial/templates/webpack/webpack.dev.js

```diff
@@ -8,12 +8,6 @@ const ChunkWebpack = webpack.optimize.CommonsChunkPlugin;
 const ContextReplacementPlugin = webpack.ContextReplacementPlugin;
 const LoaderOptionsPlugin = webpack.LoaderOptionsPlugin;
 
-const entryPoints = [
```

Comment:
I think you might've got rid of these by accident. They're needed for the HTML plugin to work properly ([see here](https://github.com/tobi-or-not-tobi/angular-librarian/blob/da76603c1d5671b4d098b362e9fea8ee7e7b5da7/commands/initial/templates/webpack/webpack.dev.js#L88))

Basically we want to ensure that the scripts get loaded in the order specified by the script, so that everything is loaded before we get to the example app.